doc/http_api.md

```diff
@@ -122,10 +123,30 @@ The POST body describes the ingestion source and parameters, such as Kafka confi
 }
 ```
 
-#### POST /api/v1/cluster/{dataset}/reassignshards
+#### POST /api/v1/cluster/{dataset}/stopshards
 
-Reassigns the shards to the given node. 
-The POST body describes the reassign shard config that should have both destination node address and the list of shards to be assigned.
+Stop all the given shards.
+The POST body describes the stop shard config that should have the list of shards to be stopped.
+
+* POST body should be a ReassignShardConfig in JSON format as follows:
+```json
+{
+    "shardList": [
```

Comment:
It's OK for the address to be empty?   I see the code parses everything as `ReassignShardConfig`.

Reply:
StopShard request does not need an address. It can stop any shards. Hence the `ReassignShardConfig` is shared between both stopShard and startShard request, we have that address object as well.